src/main/python/lbrynetservice.py

```diff
@@ -1,10 +1,15 @@
+import sys
+from twisted.internet import asyncioreactor
+if 'twisted.internet.reactor' not in sys.modules:
+    asyncioreactor.install()
```

Comment:
This might lead to confusing results if the wrong reactor was installed previously. I think it would better to either 1) just `asyncioreactor.install()` without putting it inside of a conditional or, alternately, 2) add an `else` to the existing condition which checks that the reactor installed previously is in fact an `asyncioreactor`.